src/components/ProjectSettingsTabs/GithubCommitQueue/getFormSchema.ts

```diff
@@ -0,0 +1,350 @@
+import widgets from "components/SpruceForm/Widgets";
+import { AliasRow } from "../AliasRow";
+import { overrideRadioBox, radioBoxOptions } from "../utils";
+import { FormState } from "./types";
+
+export const getFormSchema = (
+  useRepoSettings: boolean,
+  gitHubWebhooksEnabled: boolean,
+  formData: FormState,
+  repoData?: FormState
+): {
+  fields: Record<string, Field>;
+  schema: SpruceFormProps["schema"];
+  uiSchema: SpruceFormProps["uiSchema"];
+} => ({
+  fields: {},
+  schema: {
+    definitions: {
+      aliasArray: {
+        type: "array" as "array",
+        items: {
+          type: "object" as "object",
+          properties: {
+            id: {
+              type: "string" as "string",
+            },
+            alias: {
+              type: "string" as "string",
+            },
+            variant: {
+              type: "string" as "string",
+              default: "",
+            },
+            variantTags: {
+              type: "array" as "array",
+              title: "Variant Tags",
+              items: {
+                type: "string" as "string",
+                title: "Variant Tag",
+                default: "",
+              },
+            },
+            task: {
+              type: "string" as "string",
+              default: "",
+            },
+            taskTags: {
+              type: "array" as "array",
+              items: {
+                type: "string" as "string",
+                title: "Task Tag",
+                default: "",
+              },
+            },
+          },
+        },
+      },
+    },
+    type: "object" as "object",
+    properties: {
+      github: {
+        type: "object" as "object",
+        title: "GitHub",
+        properties: {
+          gitHubWebhooksEnabled: {
+            type: "null",
+            title: "GitHub Webhooks",
+            description: `GitHub webhooks ${
+              gitHubWebhooksEnabled ? "are" : "aren't"
+            } enabled.`,
+          },
+          prTestingEnabledTitle: {
+            type: "null",
+            title: "GitHub Pull Request Testing",
+            ...(useRepoSettings === undefined && {
+              description:
+                "If enabled, then untracked branches will also use the file patterns defined here for PR testing.",
+            }),
+          },
+          prTestingEnabled: {
+            type: ["boolean", "null"],
+            oneOf: radioBoxOptions(
+              ["Enabled", "Disabled"],
+              repoData?.github?.prTestingEnabled
+            ),
+          },
+          prTesting: {
+            type: "object" as "object",
+            title: "GitHub Patch Definitions",
+            description:
+              "For patches created from GitHub pull requests, Evergreen will find variants that match each variant regex, and schedule tasks that match the corresponding task regex. All regular expressions must be valid Golang regular expressions.",
```

Comment:
I realize I wrote this text 🤦‍♀️  but it should probably say "regex/tag list". We could also tweak this to be more like the later definitions:
For patches created from GitHub pull requests, Evergreen will schedule only the tasks and variants matching the tags/regexes definitions. All regular expressions must be valid Golang regular expressions.